### PR TITLE
Companion PR to substrate#4447

### PR DESCRIPTION
**Pull request by expenses**
I fucked up and didn't make a companion PR for: https://github.com/paritytech/substrate/pull/4447 before merging.

### CODE REVIEW

File: availability-store/Cargo.toml [hunk 1]
```diff
@@ -15,15 +15,15 @@ futures = "0.3.1"
 tokio = { version = "0.2.4", features = ["rt-core"] }
 exit-future = "0.2.0"
 codec = { package = "parity-scale-codec", version = "1.1.0", features = ["derive"] }
-sc-network = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
```
Comment (rphmeier) [inline availability-store/Cargo.toml]: can you point this back to `polkadot-master`?

Comment (expenses) [reply availability-store/Cargo.toml]: Just this one dep?

Comment (rphmeier) [reply availability-store/Cargo.toml]: no, all of them...

**Comment by rphmeier** [comment]
Superseded by #768

**Pull request closed without merging**